test/routes.spec.js

```diff
@@ -0,0 +1,34 @@
+import chai from 'chai';
+import server from './../src/server';
```

Comment:
Does "clean" importing work in the test Env too? Webpack should look in the 'src' folder by default (it does in the app)